index.android.js

```diff
@@ -47,8 +47,8 @@ var MusicControl = {
   resetNowPlaying: function(){
     NativeMusicControl.resetNowPlaying()
   },
-  enableControl: function(controlName, enable){
-    NativeMusicControl.enableControl(controlName, enable)
+  enableControl: function(controlName, enable, options = {}){
+    NativeMusicControl.enableControl(controlName, enable, options || {})
```

Comment:
Wouldn't adding `options = {}` and then `options || {}` be redundant?

Reply:
I'm copying what was already in index.ios.js.  I thought it was redundant as well but it also seemed weird to have a different syntax for the exact same thing in both files.